person.rb

```diff
@@ -6,8 +6,8 @@ def initialize(age, name: 'Unknown', parent_permission: true)
     @parent_permission = parent_permission
   end
 
-  attr_reader :id
   attr_accessor :name, :age
+  attr_reader :id
 
   private
 
```

Comment:
- [ ] Please consider using the or`||` logical operator instead of the and`&&` so that a person can use services when they are of age or have parent permission.
![1](https://user-images.githubusercontent.com/94551484/215855830-782b664e-7304-44f6-a3c9-9ec7b3362d24.png)